docs/healthCheck.md

```diff
@@ -0,0 +1,24 @@
+`symbol-bootstrap healthCheck`
+==============================
+
+It checks if the services created with docker compose are up and running.
+
+* [`symbol-bootstrap healthCheck`](#symbol-bootstrap-healthcheck)
+
+## `symbol-bootstrap healthCheck`
+
+It checks if the services created with docker compose are up and running.
```

Comment:
I would list exactly what things it tests. There's more information in the issue description than in this README...

Reply:
Fixed. The commands md files are generated from the TS command files.